README.md

```diff
@@ -131,21 +126,21 @@ receivers:
     address: "localhost:9411"
 
   jaeger:
-    collector_thrift_port: 14267
```

Comment:
collector_* are the correct ones for Agent, the current differences between Collector and Agent are explained in receivers/README.md